kong-ingress-data-plane-postgres.yaml

```diff
@@ -39,14 +39,18 @@ spec:
           - name: KONG_ADMIN_ERROR_LOG
             value: /dev/stderr
           - name: KONG_ADMIN_LISTEN
-            value: 'off'
+            value: 0.0.0.0:8001,0.0.0.0:8444 ssl
           - name: KONG_PROXY_LISTEN
             value: 0.0.0.0:8000,0.0.0.0:8443 ssl
         ports:
         - name: data-http
           containerPort: 8000
         - name: data-https
           containerPort: 8443
+        - name: admin-http
+          containerPort: 8001
+        - name: admin-https
+          containerPort: 8444
```

Comment:
Why is the Admin API being enabled on the data-plane Kong nodes?

Reply:
oh weird I shouldn't have been modifying this file at all ... I'll fix 👍